packages/_site/src/pages/components/popover/index.tsx

```diff
@@ -26,7 +26,7 @@ export default function PopoverPage() {
         >
             <ComponentHero
                 title="Popover"
-                description=""
+                description="A button that can be used to display some content on top of another."
```

Comment:
Use this description:
"Displays rich content in a portal, triggered by a button."